src/context/WalletProvider/KeepKey/components/RecoverySentence.tsx

```diff
@@ -1,48 +1,11 @@
 import { Alert, AlertIcon, Flex, useColorModeValue } from '@chakra-ui/react'
 import { ModalBody, ModalHeader } from '@chakra-ui/react'
-import { useToast } from '@chakra-ui/toast'
```

Comment:
Note for the reviewer: the KeepKey `reset()` is now handled in the `Label` component, as it's required to be called before `RecoverySentence.tsx` is rendered so that the PIN entry flow can be initiated.